subsys/usb/class/netusb/function_eem.c

```diff
@@ -113,12 +112,12 @@ static int eem_send(struct net_pkt *pkt)
 	*hdr = sys_cpu_to_le16(0x3FFF & len);
 	b_idx += sizeof(u16_t);
 
-	/* generate transfer buffer */
-	for (frag = pkt->frags; frag; frag = frag->frags) {
-		memcpy(&tx_buf[b_idx], frag->data, frag->len);
-		b_idx += frag->len;
+	if (net_pkt_read(pkt, &tx_buf[b_idx], net_pkt_get_len(pkt))) {
```

Comment:
This doesn't compile, I believe a `#include <net/net_pkt.h>` is missing.

Reply:
function name is wrong actually